docs/source/conf.py

```
@@ -288,11 +289,22 @@
 # -- Custom app setup --------------------------------------------------------
 
 
+def _skip_hidden_from_docs(_app, _what, _name, obj, skip):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_skip_hidden_from_docs`

Add return type annotation: `bool | None`

(ANN202)